app/src/test/java/de/westnordost/streetcomplete/overlays/street_furniture/FlagpoleMatching.kt

```diff
@@ -0,0 +1,29 @@
+package de.westnordost.streetcomplete.overlays.street_furniture
```

Comment:
as it tests `de.westnordost.streetcomplete.osm.IS_STREET_FURNITURE_INCLUDING_DISUSED_EXPRESSION`, should be renamed and moved to osm/StreetFurnitureTestKt.kt